src/main/java/com/coreos/jetcd/internal/impl/ClientUtil.java

```diff
@@ -0,0 +1,132 @@
+package com.coreos.jetcd.internal.impl;
```

Comment:
I think you might resolve merge conflict in-correctly. your pr doesn't use `SimpleEtcdNameResolverFactory` anywhere.

When you rebase your branch on master. Just remove ClientUtil.java since it is not using anywhere.